vyper/evm/opcodes.py

```diff
@@ -180,6 +180,7 @@
     "REVERT": (0xFD, 2, 0, 0),
     "INVALID": (0xFE, 0, 0, 0),
     "DEBUG": (0xA5, 1, 0, 0),
+    "BREAKPOINT": (0xA6, 0, 0, 0),
```

Comment:
is this an actual opcode used in Hardhat? I didn't think it was

Reply:
it's in titanoboa

Reply:
https://github.com/vyperlang/titanoboa/blob/fa3bc79650747a737304d459c0489ebb326ed7ca/boa/env.py#L162-L163